kse/src/org/kse/crypto/x509/ExtendedKeyUsageType.java

```diff
@@ -77,8 +77,11 @@
 	NETSCAPE_SGC("2.16.840.1.113730.4.1",  "NetscapeSGCExtKeyUsage"),
 
 	/** Microsoft Server Gated Crypto (SGC) */
-	VERISIGN_SGC("2.16.840.1.113733.1.8.1",  "VeriSignSGCExtKeyUsage");
+	VERISIGN_SGC("2.16.840.1.113733.1.8.1",  "VeriSignSGCExtKeyUsage"),
 
+	/** TLS signing specified by ETSI TS 102 231 V3.1.2 for the purpose of signing Trust-service Status Lists. */
```

Comment:
I think you misunderstood me. TSL was of course the right abbreviation for "Trusted-service Status List". 
What I meant was that "TSL" can easily be confused with "TLS" (Transport Layer Security), the successor to SSL. 

My suggestion was to make the difference more clear by writing something like "Trusted-service Status List (TSL) Signing" instead of "TSL Signing" here:

![grafik](https://user-images.githubusercontent.com/7840192/67638253-b5542a00-f8e2-11e9-9818-256e73071b52.png)